cli/src/cluster_query.rs

```diff
@@ -824,6 +832,59 @@ pub fn process_ping(
     Ok("".to_string())
 }
 
+pub fn process_live_slots(url: &str) -> ProcessResult {
+    let exit = Arc::new(AtomicBool::new(false));
+    let exit_clone = exit.clone();
+
+    ctrlc::set_handler(move || {
+        exit_clone.store(true, Ordering::Relaxed);
+    })?;
+
+    let mut current: Option<SlotInfoMessage> = None;
+    let mut message = "".to_string();
+
+    let slot_progress = new_spinner_progress_bar();
+    slot_progress.set_message("Connecting...");
+    let (mut client, receiver) = PubsubClient::slot_subscribe(url).unwrap();
+    slot_progress.set_message("Connected.");
+
+    loop {
+        if exit.load(Ordering::Relaxed) {
+            eprintln!("{}", message);
+            client.shutdown().unwrap();
+            break;
+        }
+
+        match receiver.recv() {
+            Ok(new_info) => {
+                message = format!("{:?}", new_info).to_owned();
+                slot_progress.set_message(&message);
+
+                if let Some(previous) = current {
+                    let slot_delta: i64 = new_info.slot as i64 - previous.slot as i64;
+                    let root_delta: i64 = new_info.root as i64 - previous.root as i64;
+
+                    if abs(slot_delta) != abs(root_delta) {
+                        let prev_root = format!(
+                            "|<- {} <- … <- {} <- {}",
+                            previous.root, previous.parent, previous.slot
+                        )
+                        .to_owned();
+                        slot_progress.println(&prev_root);
+                    }
+                }
```

Comment:
Can you provide an example of what this looks like in a comment on the PR?
Also, I think a code comment explaining this logic would be helpful.